.codecov.yml

```
@@ -2,7 +2,7 @@ coverage:
   status:
     project:
       default:
-        target: 80%
+        target: 70%
```

Let's keep this as high as possible

```suggestion
        target: 78%
```